community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -424,7 +428,7 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
             args = useArgumentsFromFileArgumentIfPresent( args );
 
             storeDir = args.interpretOption( Options.STORE_DIR.key(), Converters.mandatory(),
-                    Converters.toFile(), Validators.DIRECTORY_IS_WRITABLE, Validators.CONTAINS_NO_EXISTING_DATABASE );
+                    Converters.toFile(), Validators.DIRECTORY_IS_WRITABLE );
```

Comment:
not sure why this validator is gone, don't we need to check that we will not gonna import on top of some existing database?

Reply:
This is because of restartability feature which starts from an existing database. It's now not up to the ImportTool anymore to make this decision, it's up to the selected importer.. which will each do the right thing.